tempus_fugit_minecraft/model.py

```diff
@@ -529,6 +529,20 @@ def handle_jump(self) -> None:
         @returns None
         @see issue 68
         """
+     #Issue 39
```

Comment:
Please link the issue in the doxygen comment like how @bakursait2's PR has it:
`@see [Issue 39](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/39)`